### ISSUE TITLE
Name mistake in animate.css flippers (X and Y axis)

### ISSUE THREAD

**Issue by artisticLogicMK** [animate-css/animate.css#1572]
### Describe The Bug

I noticed in animate.css that the flipper animations are named incorrectly. 'X axis' is mistaken for 'Y axis.'

*flipInX and flipOutX should flip horizontally on X axis but they flip vertically on Y axis.
*flipInY and flipOutY should flip vertically on the Y axis but they flip horizontally on the X axis.

**Comment by eltonmesquita** [comment]
You are right! Thanks for catching this one, it should be fixed on v5 as this is a breaking change

**Comment by warengonzaga** [comment]
I'll take it @eltonmesquita

**Comment by warengonzaga** [comment]
PRed @eltonmesquita, thank you for the report @artisticLogicMK

**Comment by warengonzaga** [comment]
pushed a fix for v5 release #1583
